### ISSUE TITLE
support for backup only

### ISSUE THREAD

**Issue by rustyrazorblade** [protectwise/cassandra-util#5]
It's been pointed out to me recently that the backup safety net could also be used to migrate account data between clusters.  It would be preferable (and safer) to approach the problem this way:

1. do a first pass over the data to create a backup
2. restore data to second cluster
3. do a second pass to evict

Would you be interested in a PR for this option?

**Comment by MightyE** [comment]
Yeah, I definitely could see use cases for that.  Partial-tenant migrations from a multi-tenant table to a different table or cluster is a giant mess of tombstones and performance degradation today.

Would be interested in a PR for that for certain.  It does get a little weird because until now every record goes to zero or one locations, which satisfies the forward-only iterators available during compaction.  Still, I'm sure it's not hard to create a cloning iterator so that you can feed records to two sinks.
